client/src/helpers/APICalls/sendMessage.ts

```diff
@@ -0,0 +1,19 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const sendAndSavaMessage = async (conversationId: string, senderId: string, text: string) => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ conversationId, senderId, text }),
+    credentials: 'include',
+  };
+  return await fetch(`/conversation/sendMessage`, fetchOptions)
+    .then((res) => res.json())
+    .catch((err) => {
+      err: {
+        message: err;
+      }
+    });
```

Comment:
This should be:
`
.catch((err) => ({
      err: {
        message: err;
      }
    }));
`